puppetserver/Dockerfile

```diff
@@ -125,6 +125,11 @@ COPY conf.d/product.conf /etc/puppetlabs/puppetserver/conf.d/
 
 COPY puppetdb.conf /var/tmp/puppet/
 
+COPY enc/* /etc/puppetlabs/puppet
+COPY reports/* /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/reports
+RUN chmod +x /etc/puppetlabs/puppet/*.rb && \
+    chmod +x /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/reports/*.rb
```

Comment:
Is the +x flag actually required here?

Reply:
Seems so. I Get a permission error otherwise.